Tableau Strategy

olamiral-mulesoft opened the pull request:
- Added platform metrics current consolidated dashboard
- Changed tableau strategy JSON file generation rule
- Changed tableau.path property to tableau.outputDir
- Added tableau strategy request in postman collection
- Updated README.md with tableau instructions

DavoCoder reviewed (changes requested):
Since the approach is to generate a local file, this won't work in CloudHub or RTF deployment models; the file will be generated but no one will be able to access it. We need to return the payload as a downloadable file (when using the API) and/or using SFTP when using the poller

olamiral-mulesoft commented:
- Added Cloudhub Runtime Manager metrics in current consolidated Tableau dashboard
- Fixed current consolidated Tableau dashboard to look similar to Splunk version
- Changed README.md file to add instructions about limitations when MA is running on CH or RTF and how to manually generate JSON files when this is the case.
